pom.xml

```diff
@@ -20,11 +20,7 @@
 
```

Comment:
One line up from this comment should be:
```
<url>https://github.com/jenkinsci/clif-performance-testing-plugin</url>

Reply:
Good catch Daniel. I fixed that in upstream's master.